pkg/controller/configmap/configmap_controller.go

```diff
@@ -382,18 +383,19 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 		}
 	}
 
+	ncpImageChange := false
```

Comment:
`needChange.ncp` will be checked in line 441, do you think it's fine to set `ncpImageChange` instead of `ncpImageChange`

Reply:
ncpImageChange is used to check  NCP_IMAGE  or nsx-ncp replicas changed or not. Since NCP_IMAGE  is a existing of spec.template field of NCP deployment. Therefore, we don't need check it, it will be handled in line 431 ApplyObject by default.  Every time you NCP_IMAGE is updated,  k8s rolling update will be triggered after ApplyObject. They is why we don't do more check.

Reply:
needChange.ncp is for checking related section changes affecting NCP in Configmap. Althtough ApplyObject will update changes from Configmap but it won't trigger k8s starting rolling update, so we need to keep it and 
check it in line 441.

Reply:
Got it, so `ncpReplicas` change is also covered by `ApplyObject`

Reply:
Yes, I think so. Based on my test, although ncpReplicas is not  an existing field in spec.template. my testing is to done by changing **ncpReplicas: 2**
in operator.nsx.vmware.com_v1_ncpinstall_cr.yaml